2022-11-28 00:13:25,402 INFO    MainThread:8224 [wandb_setup.py:_flush():71] setting env: {}
2022-11-28 00:13:25,402 INFO    MainThread:8224 [wandb_setup.py:_flush():71] setting login settings: {}
2022-11-28 00:13:25,402 INFO    MainThread:8224 [wandb_init.py:_log_setup():371] Logging user logs to /home/josyula/Programs/MAS_Project/gym_envs_urdf/examples/neurocoevolution_pytorch/wandb/run-20221128_001325-16zd5q6n/logs/debug.log
2022-11-28 00:13:25,403 INFO    MainThread:8224 [wandb_init.py:_log_setup():372] Logging internal logs to /home/josyula/Programs/MAS_Project/gym_envs_urdf/examples/neurocoevolution_pytorch/wandb/run-20221128_001325-16zd5q6n/logs/debug-internal.log
2022-11-28 00:13:25,403 INFO    MainThread:8224 [wandb_init.py:init():404] calling init triggers
2022-11-28 00:13:25,403 INFO    MainThread:8224 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-11-28 00:13:25,403 INFO    MainThread:8224 [wandb_init.py:init():460] starting backend
2022-11-28 00:13:25,403 INFO    MainThread:8224 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-11-28 00:13:25,418 INFO    MainThread:8224 [backend.py:ensure_launched():216] starting backend process...
2022-11-28 00:13:25,423 INFO    MainThread:8224 [backend.py:ensure_launched():221] started backend process with pid: 8356
2022-11-28 00:13:25,425 INFO    MainThread:8224 [wandb_init.py:init():469] backend started and connected
2022-11-28 00:13:25,427 INFO    MainThread:8224 [wandb_init.py:init():533] updated telemetry
2022-11-28 00:13:25,443 INFO    MainThread:8224 [wandb_init.py:init():563] communicating current version
2022-11-28 00:13:26,769 INFO    MainThread:8224 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.13.5 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-11-28 00:13:26,770 INFO    MainThread:8224 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-11-28 00:13:26,935 INFO    MainThread:8224 [wandb_init.py:init():606] starting run threads in backend
2022-11-28 00:13:31,941 INFO    MainThread:8224 [wandb_run.py:_console_start():1810] atexit reg
2022-11-28 00:13:31,941 INFO    MainThread:8224 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-11-28 00:13:31,942 INFO    MainThread:8224 [wandb_run.py:_redirect():1689] Redirecting console.
2022-11-28 00:13:31,944 INFO    MainThread:8224 [wandb_run.py:_redirect():1745] Redirects installed.
2022-11-28 00:13:31,944 INFO    MainThread:8224 [wandb_init.py:init():633] run started, returning control to user process
2022-11-28 00:21:38,286 INFO    MainThread:8224 [wandb_run.py:_atexit_cleanup():1771] got exitcode: 0
2022-11-28 00:21:38,293 INFO    MainThread:8224 [wandb_run.py:_restore():1751] restore
2022-11-28 00:21:40,564 INFO    MainThread:8224 [wandb_run.py:_wait_for_finish():1908] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 879
  total_bytes: 879
}

2022-11-28 00:21:40,772 INFO    MainThread:8224 [wandb_run.py:_wait_for_finish():1908] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 879
  total_bytes: 879
}

2022-11-28 00:21:41,100 INFO    MainThread:8224 [wandb_run.py:_wait_for_finish():1908] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 879
  total_bytes: 26327
}

2022-11-28 00:21:41,202 INFO    MainThread:8224 [wandb_run.py:_wait_for_finish():1908] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 879
  total_bytes: 26327
}

2022-11-28 00:21:41,304 INFO    MainThread:8224 [wandb_run.py:_wait_for_finish():1908] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26327
  total_bytes: 26327
}

2022-11-28 00:21:41,405 INFO    MainThread:8224 [wandb_run.py:_wait_for_finish():1908] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26327
  total_bytes: 26327
}

2022-11-28 00:21:41,507 INFO    MainThread:8224 [wandb_run.py:_wait_for_finish():1908] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26327
  total_bytes: 26327
}

2022-11-28 00:21:41,608 INFO    MainThread:8224 [wandb_run.py:_wait_for_finish():1908] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26327
  total_bytes: 26327
}

2022-11-28 00:21:41,711 INFO    MainThread:8224 [wandb_run.py:_wait_for_finish():1908] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26327
  total_bytes: 26327
}

2022-11-28 00:21:42,037 INFO    MainThread:8224 [wandb_run.py:_wait_for_finish():1908] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26327
  total_bytes: 26327
}

2022-11-28 00:21:42,358 INFO    MainThread:8224 [wandb_run.py:_wait_for_finish():1908] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26327
  total_bytes: 26327
}
local_info {
}

2022-11-28 00:21:43,889 INFO    MainThread:8224 [wandb_run.py:_append_files():2174] logging synced files
